test/e2e/play_kube_test.go

```diff
@@ -1177,4 +1253,87 @@ spec:
 			Expect(inspect.OutputToString()).To(ContainSubstring(correctLabels))
 		}
 	})
+
+	It("podman play kube allows setting resource limits", func() {
+		SkipIfContainerized("Resource limits require a running systemd")
+		SkipIfNotFedora() // Requires cgroups v2
```

Comment:
SkipIfRootlessCgroupsV1()
Will this test work on a CgroupV1 rootfull environment?

Reply:
You're right. I have seen the error "invalid configuration, cannot specify resource limits without cgroups v2 and --cgroup-manager=systemd" before, but when I check the code I see that that only applies in rootless mode.

https://github.com/containers/podman/blob/0b7b22243798650841cd9b790809a0a482f4cbcb/pkg/spec/spec.go#L420-L438

Reply:
Hm, I don't understand what happened. When I had only `SkipIfContainerized` there were two failures: the rootless cgroups v1 tests. Now, with `SkipIfContainerized` _and_ `SkipIfRootlessCgroupsV1`, there are 10 CI builds that fail. :confused:

Reply:
This doesn't seem to work with Cgroups v1, even with root. All the Cgroups v1 CI builds are hitting this error:

         Error: container_linux.go:370: starting container process caused: process_linux.go:338: getting the final child's pid from pipe caused: EOF: OCI runtime error

I'm not sure if that's normal for podman and resource limits.

Reply:
Limits seem to require root and Cgroups v2. When I test with `podman run` on my local machine I get this:

```
$ bin/podman run -it --rm --cpu-quota 10000 alpine sh
Error: OCI runtime error: container_linux.go:370: starting container process caused: process_linux.go:459: container init caused: process_linux.go:422: setting cgroup config for procHooks process caused: cannot set cpu limit: container could not join or create cgroup
$ sudo bin/podman run -it --rm --cpu-quota 10000 alpine sh
[sudo] password for jordan: 
/ #
```